src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
Hmm. Yeah I suppose we could make it more multi-purpose. But the styles are specifically tailored to video embeds. What if I created a `stache-iframe` component and used it here instead? Then we could have both?